creditTest1/task1.c

```diff
@@ -0,0 +1,98 @@
+#include <stdio.h>
+#include <stdbool.h>
+#include <string.h>
+
+// #define DEBUG 1
+
+typedef enum
+{
+    OK,
+    IncorrectBinaryNumber
+} Error;
+
+Error convertNumberFromBinToOctalForm(char *numberInBinForm, char *numberInOctalForm)
+{
+    short lastIndexOfBinNumber = strlen(numberInBinForm) - 1;
+    short lastIndexOfOctalNumber = lastIndexOfBinNumber / 3 + 1;
+    short curOffset = 8;
```

Comment:
```suggestion
    short curentExponent = 8;
```
Это ведь не отступ, это степень отступа.
(-0.25)